resources-domain/resources-import-consumer/tests/unit/handler.test.ts

```diff
@@ -34,6 +35,9 @@ beforeEach(() => {
   jest.resetAllMocks();
 });
 
+const timeSequenceFromDate = (date: Date, sequence = 0): TimeSequence =>
```

Comment:
This guy gets duplicated a bit but it's only a 1 liner in test code, didn't seem worth moving into a shared place, but I can